mvrx-mocking/src/main/kotlin/com/airbnb/mvrx/mocking/MockBuilder.kt

```diff
@@ -1460,14 +1476,13 @@ open class MavericksViewMocks<V : MockableMavericksView, Args : Parcelable> @Pub
 
             numAllowedCreationsOfMocks.incrementAndGet()
 
-            val mocks =
-                view.call<MavericksViewMocks<out MockableMavericksView, out Parcelable>>("provideMocks")
+            val mocks = mockProvider.mavericksViewMocks(view)
 
             require(numAllowedCreationsOfMocks.decrementAndGet() >= 0) {
                 "numAllowedCreationsOfMocks is negative"
             }
 
-            return mocks
+            return mocks ?: DefaultViewMocksProvider.mavericksViewMocks(view)!!
```

Comment:
@digitalbuddha how about if we make `mavericksViewMocks` not have a nullable return type? If somebody has an implementation that is conditional they can manually fallback to `DefaultViewMocksProvider` since it is public. I  think this makes the contract a bit more explicit and straightforward any way

Reply:
all set! Lmk if this version works